website/content/docs/enterprise/network-segments.mdx

```diff
@@ -52,3 +66,171 @@ among themselves, and use Serf to maintain a shared view of the members of the
 pool for different purposes, like finding a Consul server in a local cluster,
 or finding servers in a remote cluster. A **segmented** LAN Gossip Pool limits a
 group of agents to only connect with the agents in its segment.
+
+# Network Segments Configuration
+
+## Server Configuration
+
+Server agents are members of all segments and, therefore, provide a connection
+point across segments. The datacenter includes a `<default>` segment, as well as
+additional segments defined in the [`segments`](/docs/agent/options#segments)
+server agent configuration option. Each additional segment is defined by:
+
+- a non-empty name
+- a unique port
+- optionally, a custom bind and advertise address for the additional segment's
+  Serf LAN listener on the server
+
+~> **Note:** Prior to Consul 1.7.3, a Consul server agent configured with too
+many network segments may not be able to start due to [limitations](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters#network-segments-limitations)
+in Serf.
+
+### Example Server Configuration
+
+The following server agent configuration will create two user-defined network
+segments: `alpha` and `beta`.
+
+<CodeTabs heading="Example network segments configuration for server agents">
+
+```hcl
+segments = [
+  {
+    name = "alpha"
+    bind = "{{GetPrivateIP}}"
+    advertise = "{{GetPrivateIP}}"
+    port = 8303
+  },
+  {
+    name = "beta"
+    bind = "{{GetPrivateIP}}"
+    advertise = "{{GetPrivateIP}}"
+    port = 8304
+  }
+]
+```
+
+```json
+{
+  "segments": [
+    {
+      "name": "alpha",
+      "bind": "{{GetPrivateIP}}",
+      "advertise": "{{GetPrivateIP}}",
+      "port": 8303
+    },
+    {
+      "name": "beta",
+      "bind": "{{GetPrivateIP}}",
+      "advertise": "{{GetPrivateIP}}",
+      "port": 8304
+    }
+  ]
+}
+```
+
+</CodeTabs>
+
+The server [agent configuration](/docs/agent/options) options relevant to network
+segments are:
+
+- [`ports.serf_lan`](/docs/agent/options#serf_lan_port): The Serf LAN port on this server
+  for the `<default>` network segment's gossip pool.
+- [`segments`](/docs/agent/options#segments): A list of user-defined network segments
+  on this server, including their names and Serf LAN ports.
+
+## Client Configuration
+
+Each client agent can only be a member of one segment at a time. This will be the
+`<default>` segment unless otherwise specified in their [`segment`](/docs/agent/options.html#_segment)
+agent configuration option.
```

Comment:
```suggestion
`<default>` segment unless otherwise specified in the agent's [`segment`](/docs/agent/options.html#_segment) configuration option.
```